gitlint/rules.py

```diff
@@ -79,14 +81,14 @@ def __eq__(self, other):
         return equal
 
     def __str__(self):
-        return sstr(self)  # pragma: no cover
+        return self.__unicode__()  # pragma: no cover
```

Comment:
Why do you use `self.__unicode__()`? Does this work with Python 2 too?

Reply:
Yes, this works also with python 2 (Tested on python 2.7.14)